pkg/apis/core/types.go

```diff
@@ -5494,6 +5507,15 @@ type PodLogOptions struct {
 	// the actual log data coming from the real kubelet).
 	// +optional
 	InsecureSkipTLSVerifyBackend bool
+
+	// Specify which container log stream to return to the client.
+	// Acceptable values are "All", "stdout" and "stderr". If not specified, "All" is used, and both stdout and stderr
+	// are returned interleaved.
+	// Note that when "TailLines" is specified, "Stream" can only be set to "All" for now.
```

Comment:
```suggestion
	// Note that when "TailLines" is specified, "Stream" can only be set to "All".
```

let's not promise anything in the description